front/app/modules/commercial/insights/admin/containers/Insights/Edit/Edit.test.tsx

```diff
@@ -81,7 +131,10 @@ describe('Insights Edit', () => {
         fireEvent.click(screen.getByText('+'));
       });
 
-      expect(spy).toHaveBeenCalledWith(viewId, categoryName);
+      expect(service.addInsightsCategory).toHaveBeenCalledWith(
```

Comment:
Ah, cool, it automatically spies on mocked methods then ! Good to know.

Reply:
Yees! I also discovered this recently, pretty cool :)